2018-10-19 21:39:13,354 ERROR: Exception on /user/djj [GET] [in D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1811, in full_dispatch_request
    rv = self.preprocess_request()
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 2087, in preprocess_request
    rv = func()
  File "D:\PycharmProjects\FlaskWeb\app\routes.py", line 28, in before_request
    g.locale = str(get_locale())
  File "D:\Anaconda3\lib\site-packages\flask_babel\__init__.py", line 261, in get_locale
    locale = Locale.parse(rv)
  File "D:\Anaconda3\lib\site-packages\babel\core.py", line 268, in parse
    parts = parse_locale(identifier, sep=sep)
  File "D:\Anaconda3\lib\site-packages\babel\core.py", line 1094, in parse_locale
    raise ValueError('expected only letters, got %r' % lang)
ValueError: expected only letters, got 'zh-tw'
2018-10-19 21:39:20,866 ERROR: Exception on /favicon.ico [GET] [in D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1811, in full_dispatch_request
    rv = self.preprocess_request()
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 2087, in preprocess_request
    rv = func()
  File "D:\PycharmProjects\FlaskWeb\app\routes.py", line 28, in before_request
    g.locale = str(get_locale())
  File "D:\Anaconda3\lib\site-packages\flask_babel\__init__.py", line 261, in get_locale
    locale = Locale.parse(rv)
  File "D:\Anaconda3\lib\site-packages\babel\core.py", line 268, in parse
    parts = parse_locale(identifier, sep=sep)
  File "D:\Anaconda3\lib\site-packages\babel\core.py", line 1094, in parse_locale
    raise ValueError('expected only letters, got %r' % lang)
ValueError: expected only letters, got 'zh-tw'
2018-10-19 21:39:23,200 ERROR: Exception on / [GET] [in D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1811, in full_dispatch_request
    rv = self.preprocess_request()
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 2087, in preprocess_request
    rv = func()
  File "D:\PycharmProjects\FlaskWeb\app\routes.py", line 28, in before_request
    g.locale = str(get_locale())
  File "D:\Anaconda3\lib\site-packages\flask_babel\__init__.py", line 261, in get_locale
    locale = Locale.parse(rv)
  File "D:\Anaconda3\lib\site-packages\babel\core.py", line 268, in parse
    parts = parse_locale(identifier, sep=sep)
  File "D:\Anaconda3\lib\site-packages\babel\core.py", line 1094, in parse_locale
    raise ValueError('expected only letters, got %r' % lang)
ValueError: expected only letters, got 'zh-tw'
2018-10-19 21:39:27,076 ERROR: Exception on /favicon.ico [GET] [in D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1811, in full_dispatch_request
    rv = self.preprocess_request()
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 2087, in preprocess_request
    rv = func()
  File "D:\PycharmProjects\FlaskWeb\app\routes.py", line 28, in before_request
    g.locale = str(get_locale())
  File "D:\Anaconda3\lib\site-packages\flask_babel\__init__.py", line 261, in get_locale
    locale = Locale.parse(rv)
  File "D:\Anaconda3\lib\site-packages\babel\core.py", line 268, in parse
    parts = parse_locale(identifier, sep=sep)
  File "D:\Anaconda3\lib\site-packages\babel\core.py", line 1094, in parse_locale
    raise ValueError('expected only letters, got %r' % lang)
ValueError: expected only letters, got 'zh-tw'
2018-10-19 22:23:00,017 INFO: Yuan_blog startup [in D:\PycharmProjects\FlaskWeb\app\__init__.py:63]
2018-10-19 22:25:54,509 ERROR: Exception on / [GET] [in D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1811, in full_dispatch_request
    rv = self.preprocess_request()
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 2087, in preprocess_request
    rv = func()
  File "D:\PycharmProjects\FlaskWeb\app\routes.py", line 28, in before_request
    g.locale = str(get_locale())
  File "D:\Anaconda3\lib\site-packages\flask_babel\__init__.py", line 261, in get_locale
    locale = Locale.parse(rv)
  File "D:\Anaconda3\lib\site-packages\babel\core.py", line 268, in parse
    parts = parse_locale(identifier, sep=sep)
  File "D:\Anaconda3\lib\site-packages\babel\core.py", line 1094, in parse_locale
    raise ValueError('expected only letters, got %r' % lang)
ValueError: expected only letters, got 'zh-tw'
2018-10-19 22:26:11,009 ERROR: Exception on /favicon.ico [GET] [in D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py:1761]
Traceback (most recent call last):
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\_compat.py", line 35, in reraise
    raise value
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 1811, in full_dispatch_request
    rv = self.preprocess_request()
  File "D:\PycharmProjects\FlaskWeb\venv\lib\site-packages\flask\app.py", line 2087, in preprocess_request
    rv = func()
  File "D:\PycharmProjects\FlaskWeb\app\routes.py", line 28, in before_request
    g.locale = str(get_locale())
  File "D:\Anaconda3\lib\site-packages\flask_babel\__init__.py", line 261, in get_locale
    locale = Locale.parse(rv)
  File "D:\Anaconda3\lib\site-packages\babel\core.py", line 268, in parse
    parts = parse_locale(identifier, sep=sep)
  File "D:\Anaconda3\lib\site-packages\babel\core.py", line 1094, in parse_locale
    raise ValueError('expected only letters, got %r' % lang)
ValueError: expected only letters, got 'zh-tw'
2018-10-19 22:27:42,848 INFO: Yuan_blog startup [in D:\PycharmProjects\FlaskWeb\app\__init__.py:63]
